backend/DAO/studentDAO.js

```diff
@@ -58,15 +58,22 @@ module.exports = {
             resolve(studentToSkills);
         });
     },
-    // getStudentsBySkills: (skillsId) => {
-    //     return database('student')
-    //         .select(['student.id', 'student.name'])
-    //         .leftJoin('student_has_skills AS shs', 'shs.student_id', 'student.id')
-    //         .whereIn('shs.skill_id', skillsId)
-    //         .groupBy('student.id')
```

Comment:
let's remove comments like this one if they are not useful anymore

Reply:
Those lines where deleted